packages/reports/addon/consumers/request/sort.js

```diff
@@ -46,16 +46,42 @@ export default ActionConsumer.extend({
 
       request.removeSortByMetricName(metricName);
     },
+    /**
```

Comment:
Make sure you update copyright in changed files.

Reply:
The year?

Reply:
Yeah, whenever we update a file, just bump the year to 2018